ios/FluentUI.Demo/FluentUI.Demo/Demos/ListItemDemoController_SwiftUI.swift

```diff
@@ -146,60 +148,66 @@ struct ListItemDemoView: View {
+                .footerLineLimit(footerLineLimit)
+                .combineTrailingContentAccessibilityElement(trailingContentFocusableElementCount < 2)
+                .onAccessoryTapped {
+                    showingSecondaryAlert = true
+                }
+            listItem
+                .overrideTokens($overrideTokens.wrappedValue ? listItemTokenOverrides : [:])
+            return listItem
+        }
+
         @ViewBuilder
         var listItem: some View {
-            ListItem(title: title,
-                     subtitle: showSubtitle ? subtitle : "",
-                     footer: showFooter ? footer : "",
-                     leadingContent: {
-                         if showLeadingContent {
-                             leadingContent
-                         }
-                     },
-                     trailingContent: {
-                        if showTrailingContent {
-                             switch trailingContentFocusableElementCount {
-                             case 0:
-                                 Text("Spreadsheet")
-                             case 1:
-                                 Toggle("", isOn: $trailingContentToggleEnabled)
-                             default:
-                                 HStack {
-                                     Button {
-                                         showingSecondaryAlert = true
-                                     } label: {
-                                         Text("Button 1")
-                                     }
-                                     Button {
-                                         showingSecondaryAlert = true
-                                     } label: {
-                                         Text("Button 2")
-                                     }
-                                 }
-                             }
-                         }
-                     },
-                     action: !isTappable ? nil : {
-                         showingPrimaryAlert = true
-                     }
-            )
-            .backgroundStyleType(backgroundStyle)
-            .accessoryType(accessoryType)
-            .leadingContentSize(leadingContentSize)
-            .titleLineLimit(titleLineLimit)
-            .subtitleLineLimit(subtitleLineLimit)
-            .footerLineLimit(footerLineLimit)
-            .combineTrailingContentAccessibilityElement(trailingContentFocusableElementCount < 2)
-            .onAccessoryTapped {
-                showingSecondaryAlert = true
-            }
-            .disabled(isDisabled)
-            .alert("List Item tapped", isPresented: $showingPrimaryAlert) {
-                Button("OK", role: .cancel) { }
-            }
-            .alert("Detail button tapped", isPresented: $showingSecondaryAlert) {
-                Button("OK", role: .cancel) { }
-            }
+            overridenListItem()
+                .disabled(isDisabled)
```

Comment:
these modifiers are kinda tricky. since they're not in `ListItemModifiers`, applying them on `listItem` before we apply `overrideTokens` changes `listItem` from type `ListItem` to type `some View` so we need to apply them after `overrideTokens`. however we can't directly apply them after `overrideTokens` because the type that returns has no member `.disabled`/`.alert` so I had to move it to a separate view builder. this really is not that clean so if I'm missing something obvious let me know.

Reply:
Mentioned the issue in your other comment: the `overrideTokens` function should return `Self`.

The modifying function on `FluentButton[Toggle]Style` is a little different because it's not meant to be chained--that might be something we investigate and change later as well.